queries/cmake/highlights.scm

```diff
@@ -0,0 +1,101 @@
+[
+ (bracket_content)
+ (quoted_element)
+ (unquoted_argument)
+] @parameter
```

Comment:
for some reason everything is highlighted as parameter. We usually highlight parameters only in a function definition.